### ISSUE TITLE
Crouton: command not found

### ISSUE THREAD

**Issue by Nocturnalsingularity** [dnschneid/crouton#4731]
Ive set my Chromebook to dev mode. Debugged it too, set password, and sudo password works too. Im doing something wrong though since i follow the instructions and keep on getting command not found after typing sudo crouton -t xfce

**Comment by TomTravis** [comment]
in  the shell cd  /usr/local/bin the file crouton should be installed
if  not it missing

From README
 If you're just here to use crouton, you can grab the latest release from https://goo.gl/fd3zc. Download it, pop open a shell (Ctrl+Alt+T, type shell and hit enter), make the installer executable with sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton, then launch it with sudo crouton to see the help text. See the "examples" section for some usage examples.

If you're modifying crouton, you'll probably want to clone or download the repo into a subdirectory of /usr/local and then either run installer/main.sh directly, or use make to build your very own crouton. You can also download the latest release, install it as above and run crouton -x to extract out the juicy scripts contained within, but you'll be missing build-time stuff like the Makefile. You also need to remember to place the unbundled scripts somewhere in /usr/local in order to be able to execute them.

**Comment by Nocturnalsingularity** [comment]
I did this. I did everything the way the instructions explain and keep getting the same command not found. I see that cruton is installed, but im unable to run it for some reason.

**Comment by Nocturnalsingularity** [comment]
Okay now im told install: target '/usr/local/bin is not a directory

**Comment by TomTravis** [comment]
yes I it not as point and click, It can be complex to some people, so just calm down it a machine that all it is
error are part of it
you have post you data word of Mount can be difficult to work with I assume you know how to cut and paste and take screen captures, please do this, I can tell you commands but I have a feeling this is not helpful, this is from a Intell i5 debian install the has been running for 10 yrs

coco@coco:~$ ping github.com
PING github.com (192.30.255.113) 56(84) bytes of data.
64 bytes from lb-192-30-255-113-sea.github.com (192.30.255.113): icmp_seq=1 ttl=44 time=37.8 ms
64 bytes from lb-192-30-255-113-sea.github.com (192.30.255.113): icmp_seq=2 ttl=44 time=39.6 ms
64 bytes from lb-192-30-255-113-sea.github.com (192.30.255.113): icmp_seq=3 ttl=44 time=39.5 ms
^C
--- github.com ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 6ms
rtt min/avg/max/mdev = 37.832/38.986/39.585/0.847 ms
coco@coco:~$ whois ^C
coco@coco:~$ ^C
coco@coco:~$ whois 192.30.255.113

#
# ARIN WHOIS data and services are subject to the Terms of Use
# available at: https://www.arin.net/resources/registry/whois/tou/
#
# If you see inaccuracies in the results, please report at
# https://www.arin.net/resources/registry/whois/inaccuracy_reporting/
#
# Copyright 1997-2022, American Registry for Internet Numbers, Ltd.
#


NetRange:       192.30.252.0 - 192.30.255.255
CIDR:           192.30.252.0/22
NetName:        GITHUB-NET4-1
NetHandle:      NET-192-30-252-0-1
Parent:         NET192 (NET-192-0-0-0-0)
NetType:        Direct Allocation
OriginAS:       AS36459
Organization:   GitHub, Inc. (GITHU)
RegDate:        2012-11-15
Updated:        2021-12-14
Ref:            https://rdap.arin.net/registry/ip/192.30.252.0


OrgName:        GitHub, Inc.
OrgId:          GITHU
Address:        88 Colin P Kelly Jr Street
City:           San Francisco
StateProv:      CA
PostalCode:     94107
Country:        US
RegDate:        2012-10-22
Updated:        2021-05-20
Comment:        https://github.com
Comment:        Please contact us directly for matters pertaining to abuse.
Comment:        Urgent matters including DDoS are handled 24x7.
Ref:            https://rdap.arin.net/registry/entity/GITHU


OrgNOCHandle: GITHU-ARIN
OrgNOCName:   GitHub Ops
OrgNOCPhone:  +1-415-735-4488 
OrgNOCEmail:  hostmaster@github.com
OrgNOCRef:    https://rdap.arin.net/registry/entity/GITHU-ARIN

OrgAbuseHandle: GITHU1-ARIN
OrgAbuseName:   GitHub Abuse
OrgAbusePhone:  +1-415-857-5430 
OrgAbuseEmail:  abuse@github.com
OrgAbuseRef:    https://rdap.arin.net/registry/entity/GITHU1-ARIN

OrgTechHandle: GITHU-ARIN
OrgTechName:   GitHub Ops
OrgTechPhone:  +1-415-735-4488 
OrgTechEmail:  hostmaster@github.com
OrgTechRef:    https://rdap.arin.net/registry/entity/GITHU-ARIN


#
# ARIN WHOIS data and services are subject to the Terms of Use
# available at: https://www.arin.net/resources/registry/whois/tou/
#
# If you see inaccuracies in the results, please report at
# https://www.arin.net/resources/registry/whois/inaccuracy_reporting/
#
# Copyright 1997-2022, American Registry for Internet Numbers, Ltd.
#
![git](https://user-images.githubusercontent.com/46386261/168090395-5ae2aa71-ef02-4938-b409-df192d748ee8.png)

**Comment by TomTravis** [comment]
ok i just checked new powerwashed chromebook the shell has not usr/local/bin dir

this command creates the dir
install -Dt /usr/local/bin -m 755 ~/Downloads/crouton
installs the files and creates the dir

**Comment by Nocturnalsingularity** [comment]
Yeah, i keep doing that and sometimes after -DT ~/Downloads/cruton -m 755
/usr/local/bin the output is nothing, just jumps me right directly down,
outputting no txt at all, then other times i type this in and get output
command not found DT
Or sudo: command: -DT not found. I will do it again after i get home in a
few, and can actually type this in to see again exsactly what outputs

**Comment by TomTravis** [comment]
it case upper and lower install  -Dt works
chronos@localhost / $ sudo enter-chroot -c /media/removable/sd20^C                
chronos@localhost / $ install -dt /usr/local/bin -m 755 ~/Downloads/startj.sh
install: target directory not allowed when installing a directory
chronos@localhost / $ install -dt /usr/local/bin -m 755 ~/Downloads/startj.sh^C
chronos@localhost / $ install /usr/local/bin -m 755 ~/Downloads/startj.sh
install: omitting directory '/usr/local/bin'
chronos@localhost / $ install -t /usr/local/bin -m 755 ~/Downloads/startj.sh
install: cannot stat '/home/chronos/user/Downloads/startj.sh': No such file or directory
chronos@localhost / $ install -dt /usr/local/bin -m 755 ~/Downloads/startj.sh^C
chronos@localhost / $ install -Dt /usr/local/bin -m 755 ~/Downloads/startj.sh
install: cannot stat '/home/chronos/user/Downloads/startj.sh': No such file or directory
chronos@localhost / $ install -dt /usr/local/bin -m 755 ~/Downloads/startj.sh
install: target directory not allowed when installing a directory
chronos@localhost / $ install -Dt /usr/local/bin -m 755 ~/Downloads/startj.sh
install: cannot create regular file '/usr/local/bin/startj.sh': Permission denied
chronos@localhost / $ 
chronos@localhost / $ sudo !!
sudo install -Dt /usr/local/bin -m 755 ~/Downloads/startj.sh
chronos@localhost / $

**Comment by TomTravis** [comment]
it a machine you have to use the correct commands
